.travis.yml

```diff
@@ -4,16 +4,17 @@ language: node_js
 node_js:
   - "lts/*"
 cache:
-  directories:
-    - "node_modules"
+  yarn: true
 
 env:
   global:
     - AWS_REGION=us-east-1
     - SLS_DEBUG=true
 
 before_install:
-  - npm install -g serverless
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.6.0
```

Comment:
I was going complain about this until I noticed that this is the recommended way according to the Yarn doco. Fetching shell scripts without even verifying a checksum, and executing the command, what could possibly go wrong 😆